src/overlays/actors/ovl_En_Dekubaba/z_en_dekubaba.c

```diff
@@ -44,188 +54,1324 @@ const ActorInit En_Dekubaba_InitVars = {
+    /* Goron punch    */ DMG_ENTRY(2, DEKUBABA_DMGEFF_NONE),
+    /* Sword          */ DMG_ENTRY(1, DEKUBABA_DMGEFF_CUT),
+    /* Goron pound    */ DMG_ENTRY(4, DEKUBABA_DMGEFF_NONE),
+    /* Fire arrow     */ DMG_ENTRY(1, DEKUBABA_DMGEFF_FIRE),
+    /* Ice arrow      */ DMG_ENTRY(3, DEKUBABA_DMGEFF_ICE),
+    /* Light arrow    */ DMG_ENTRY(3, DEKUBABA_DMGEFF_LIGHT),
+    /* Goron spikes   */ DMG_ENTRY(1, DEKUBABA_DMGEFF_NONE),
+    /* Deku spin      */ DMG_ENTRY(1, DEKUBABA_DMGEFF_CUT),
+    /* Deku bubble    */ DMG_ENTRY(3, DEKUBABA_DMGEFF_NONE),
+    /* Deku launch    */ DMG_ENTRY(2, DEKUBABA_DMGEFF_NONE),
+    /* UNK_DMG_0x12   */ DMG_ENTRY(0, DEKUBABA_DMGEFF_NUT),
+    /* Zora barrier   */ DMG_ENTRY(0, DEKUBABA_DMGEFF_ELECTRIC),
+    /* Normal shield  */ DMG_ENTRY(0, DEKUBABA_DMGEFF_NONE),
+    /* Light ray      */ DMG_ENTRY(0, DEKUBABA_DMGEFF_NONE),
+    /* Thrown object  */ DMG_ENTRY(1, DEKUBABA_DMGEFF_NONE),
+    /* Zora punch     */ DMG_ENTRY(1, DEKUBABA_DMGEFF_NONE),
+    /* Spin attack    */ DMG_ENTRY(1, DEKUBABA_DMGEFF_CUT),
+    /* Sword beam     */ DMG_ENTRY(0, DEKUBABA_DMGEFF_NONE),
+    /* Normal Roll    */ DMG_ENTRY(0, DEKUBABA_DMGEFF_NONE),
+    /* UNK_DMG_0x1B   */ DMG_ENTRY(0, DEKUBABA_DMGEFF_NONE),
+    /* UNK_DMG_0x1C   */ DMG_ENTRY(0, DEKUBABA_DMGEFF_NONE),
+    /* Unblockable    */ DMG_ENTRY(0, DEKUBABA_DMGEFF_NONE),
+    /* UNK_DMG_0x1E   */ DMG_ENTRY(0, DEKUBABA_DMGEFF_NONE),
+    /* Powder Keg     */ DMG_ENTRY(1, DEKUBABA_DMGEFF_NONE),
 };
 
-// static InitChainEntry sInitChain[] = {
-static InitChainEntry D_808B4E94[] = {
+static InitChainEntry sInitChain[] = {
     ICHAIN_F32(targetArrowOffset, 1500, ICHAIN_STOP),
 };
 
-#endif
+void EnDekubaba_Init(Actor* thisx, PlayState* play) {
+    EnDekubaba* this = THIS;
+    s32 i;
+
+    Actor_ProcessInitChain(&this->actor, sInitChain);
+    ActorShape_Init(&this->actor.shape, 0.0f, ActorShadow_DrawCircle, 22.0f);
+    SkelAnime_Init(play, &this->skelAnime, &gDekuBabaSkel, &gDekuBabaFastChompAnim, this->jointTable, this->morphTable,
+                   DEKUBABA_LIMB_MAX);
+    Collider_InitAndSetJntSph(play, &this->collider, &this->actor, &sJntSphInit, this->colliderElements);
+    CollisionCheck_SetInfo(&this->actor.colChkInfo, &sDamageTable, &sColChkInfoInit);
+
+    if (this->actor.params == DEKUBABA_BIG) {
+        this->size = 2.5f;
+
+        for (i = 0; i < sJntSphInit.count; i++) {
+            this->collider.elements[i].dim.worldSphere.radius = this->collider.elements[i].dim.modelSphere.radius =
+                (sJntSphElementsInit[i].dim.modelSphere.radius * 2.50f);
+        }
+
+        this->actor.colChkInfo.health = 4;
+        this->actor.hintId = 8;
+        this->actor.targetMode = 2;
+    } else {
+        this->size = 1.0f;
+
+        for (i = 0; i < sJntSphInit.count; i++) {
+            this->collider.elements[i].dim.worldSphere.radius = this->collider.elements[i].dim.modelSphere.radius;
+        }
+        this->actor.hintId = 7;
```

Comment:
This needs to be updated with the TATL_HINT_ID enum now, since the PR was merged.